content/en/docs/spinnaker/using-dinghy.md

```diff
@@ -451,6 +451,162 @@ Note how the `requisiteStageRefIds` is overwritten while calling the module so t
 
 ![](/images/Screen_Shot_2018-03-26_at_5_06_25_PM.png)
 
+## Local module functionality
```

Comment:
What's benefit to local modules/ what problem does it solve? We want to start to add that context so people know why they should use something. Maybe @akshayabd can help out here? Aimee is on PTO, and I have 0 context for Dinghy.

Reply:
This was one functionality suggested by THD, https://github.com/armory/dinghy/issues/88

Reply:
Thanks! I'll add the info Monday.